typescript/infra/scripts/middleware/portal-relayer.ts

```diff
@@ -39,16 +40,27 @@ async function relayPortalTransfers() {
 
   while (true) {
     for (const chain of Object.keys(bridgeAdapterConfigs)) {
+      console.log('chain', chain);
+
       const txHashes = await app.fetchPortalBridgeTransactions(chain);
       const portalMessages = (
         await Promise.all(
           txHashes.map((txHash) => app.parsePortalMessages(chain, txHash)),
         )
       ).flat();
 
+      console.log('Portal messages', portalMessages);
```

Comment:
I know I added this but consider using the `log` from `@hyperlane-xyz/utils` so that this doesn't take up tons of lines